readme.adoc

```diff
@@ -10,86 +10,16 @@ The project is composed by 2 parts:
 == Installation
 
 Build locally
-// todo release
+// TODO release
 
 ----
 mvn clean install
 ----
 
-2. Copy `target/neo4j-kafka-*.jar` into `$NEO4J_HOME/plugins`
+2. Copy `<producer_or_consumer_dir>/target/neo4j-streams-*.jar` into `$NEO4J_HOME/plugins`
 3. Restart Neo4j
 
-== Quick Configuration
```

Comment:
Where does the user see the quick config in the readme?
I wouldn't remove it.

Reply:
Added to links in order to don't made the same changes on different files ok?